onnx/helper.py

```diff
@@ -1533,3 +1533,34 @@ def get_all_tensor_dtypes() -> KeysView[int]:
     :return: all tensor types from TensorProto
     """
     return mapping.TENSOR_TYPE_MAP.keys()
+
+
+attribute_proto_type_to_str = {
+    AttributeProto.FLOAT: "float",
```

Comment:
```suggestion
    AttributeProto.FLOAT: "FLOAT",
```

I would use upper case for the string for ONNX naming consistency.